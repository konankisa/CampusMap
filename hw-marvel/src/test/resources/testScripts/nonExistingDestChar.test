# Test to see if a non-existing/incorrect caps
# destination character throws an error

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK
AddNode characters VENUS-II
AddEdge characters GORILLA-MAN HAWK AVF-4
AddEdge characters HAWK VENUS-II AVF-5

FindPath characters GORILLA-MAN VENUS-II

FindPath characters GORILLA-MAN TESTER-MAN

FindPath characters gorilla-man HAWK