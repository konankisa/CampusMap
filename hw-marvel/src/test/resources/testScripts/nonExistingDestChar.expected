# Test to see if a non-existing/incorrect caps
# destination character throws an error

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters
added node VENUS-II to characters
added edge AVF-4 from GORILLA-MAN to HAWK in characters
added edge AVF-5 from HAWK to VENUS-II in characters

path from GORILLA-MAN to VENUS-II:
GORILLA-MAN to HAWK via AVF-4
HAWK to VENUS-II via AVF-5

unknown: TESTER-MAN

unknown: gorilla-man